VideoAPI/VideoApi.DAL/Migrations/20200429133402_CreateConferenceTableIdexes.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VideoApi.DAL.Migrations
+{
+    public partial class CreateConferenceTableIdexes : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateIndex(
+                name: "IX_Conference_HearingRefId",
+                table: "Conference",
+                column: "HearingRefId");
+            
+            migrationBuilder.CreateIndex(
+                name: "IX_Conference_State",
+                table: "Conference",
+                column: "State");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
```

Comment:
Hi Atif, Should be one index for hearingrefid + status.

Reply:
I have Changed it